docs/TROUBLE.md

```diff
@@ -0,0 +1,175 @@
+
+When the Tx module and receiver are bound and powered on, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+You should also use the [mLRS LUA script](LUA.md) to confirm the radio link and firmware versions.  It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.
+
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+
+- If the binding procedure doesn't establish the connection, verify that both the receiver and Tx module are compatible, which means they must be on the same band (2.4 GHz, 915/866 MHz or 433 MHz). In the case of 915/866 MHz, also check that the LoRa chipsets are [compatible](SX126x_SX127x_INCOMPATIBILITY.md).  For example, the FrSky R9 series and ELRS 900 MHz receivers are not compatible with the MatekSys 900 MHz hardware.
+
+- Be sure you have flashed the correct [firmware file](https://github.com/olliw42/mLRS#firmware-flashing).  Files which start with tx are only for modules functioning in the Tx role and files which start with rx are only for modules functioning in the receiver role.  Also, pay attention to the variants for different use cases and hardware configurations.  For example, there are 3 separate files for different hardware and use case options for the MatekSys mR900-30 when used as a Tx module and a 4th file for when the same hardware is used as a Rx.  A module with rx firmware flashed won't connect with any other module with rx firmware flashed and tx firmware won't connect with tx firmware.
+
+## 4. Confirm MAVLink message flow from flight controller to receiver and Tx module
+
+In EdgeTX/OpenTX go to MDL->TELEMETRY and select "Discover new sensors".  Then, scroll up using the scroll wheel and you should find a sensor named FM among several other sensors.  It should have a star flashing next to it indicating it is updating about once per second.  See the [CRSF](CRSF.md] documentation page for background info.
+
+Above, you have already confirmed the radio link and communication between the Tx module and OpenTX/EdgeTX.  So, if you don't see the FM sensor updating, this very likely indicates that the HEARTBEAT MAVLink message which is sent once per second by ArduPilot on all MAVLink interfaces is not reaching the Rx.
+
+- Check your serial port wiring between the flight controller and the receiver to ensure you have connected serial rx on one board to serial tx on the other board in both directions and ground to ground.
+
+- Use the mLRS LUA script or another configuration method to check your [mLRS receiver setup](CRSF.md#mlrs-rx-module-setup) to be sure MAVLink is selected for the Rx Ser Link Mode and that [Rx Ser Baudrate](PARAMETERS.md#rx-ser-baudrate) is correct and matches what you set in the following bullet.
```

Comment:
- 'Lua' not 'LUA'
- MAVLinkX is now the default so '...be sure MAVLinkX or MAVLinkx...'